src/docs/uxd/alertsNotifications/DOC.md

```diff
@@ -1,5 +1,87 @@
 # Alerts and Notifications
 
-Content TBD
+This document is an extension of the [Patternfly documentation on Alerts and Notifications](https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/alerts-and-notifications).
 
-[Google Doc](https://docs.google.com/document/d/1z-xPgVYb3R7tqrxAWSiEsOwB_qjpeWgLrKKShVylguE/edit?usp=sharing)
+## Modal dialog processes
+
+After completing an action or process in a modal dialog, the user should get an inline notification informing them of the result.
+(If a modal dialog has more than 2 screens, follow the pattern defined for wizards)
+![Inline notification for an error](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/uxd/img/errorPlacement.png?raw=true)
+*Error placement*
+
+### Exiting modals
+
+Modals that have only 1 step, and are fairly simple can be cancelled in a few ways:
+
+- Clicking outside of the modal
+- Clicking the "X" in the upper right corner
+- Clicking "cancel"
+
+Modals that are more complex, or are wizards, can be cancelled in the following ways:
+
+- Clicking the "X" in the upper right corner
+- Clicking "Cancel"
+- NOT clicking outside the modal
+
+In some cases, you can use a cancellation alert to inform the user that they will lose their data if cancelling the process
+![Example of cancellation alert](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/uxd/img/cancellationAlert.png?raw=true)
+*Example of cancellation alert*
+
+## Wizards
+
+Wizards are multiple step processes. In RHCS, these mostly occur in modal dialogs. At the end of a wizard, show a confirmation screen.
+![Wizard with confirmation screen](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/uxd/img/wizardConfirmation.png?raw=true)
+*Wizard with confirmation screen*
+
+After the confirmation screen. there are a few different options:
+
+**A process that does not take significant time**
+Close modal and show results. Show inline notification (see below for more details)
+If failure, see error state below.
+
+**A process that takes time, and must complete before proceeding**
+Show progress indicator as an additional screen in the modal.
+At the end of the progress, close the modal automatically and show an inline notification (see below for more information)
```

Comment:
"end of the progress" -> "end of the process"?

Reply:
it should read progress indicator

It's referencing the step above